src/resolve-dependency.js

```diff
@@ -5,10 +5,13 @@ const { isAbsolute, resolve, sep } = require('path');
 // (package.json files are emitted as they are hit)
 module.exports = function resolveDependency (specifier, parent, job) {
   let resolved;
-  if (isAbsolute(specifier) || specifier === '.' || specifier === '..' || specifier.startsWith('./') || specifier.startsWith('../'))
-    resolved = resolvePath(resolve(parent, '..', specifier), parent, job);
-  else
+  if (isAbsolute(specifier) || specifier === '.' || specifier === '..' || specifier.startsWith('./') || specifier.startsWith('../')) {
+    const trailingSlash = specifier[specifier.length - 1] === '/' || specifier[specifier.length - 1] === '\\';
```

Comment:
```suggestion
    const trailingSlash = specifier.endsWith('/') || specifier.endsWith('\\');
```

Also, I don't see a unit test for the backslash (`\\`) use case?

Reply:
I double-checked in Node.js and it doesn't support a tailing `\`... so I've removed this test entirely.